### DISCUSSION TITLE
"Move bed to front after cancelling a print" not functioning

### DISCUSSION THREAD

**Discussion by Brereton79** [mriscoc/Ender3V2S1#1058] [General]
Ender 3v2
Board 4.2.7
Firmware 20230805

When I cancel a print the bed moves to the back, not front. All other features and settings are functioning and correct. When the print finishes properly the bed parks to the front as it should.

-Swede.

**Comment by mriscoc** [comment]
Currently the firmware follow the standard Marlin behavior after a print cancel:
```cpp
  #define EVENT_GCODE_SD_ABORT "G28XY"      // G-code to run on SD Abort Print (e.g., "G28XY" or "G27")
```
If you want other g-code you can change that line in the configuration_adv.h file and make a custom build.

**Comment by Brereton79** [comment]
Ok. I thought it was supposed to move to the front. Not a major issue I was wondering if it was a setting or something I messed up. Thanks for the reply.
-Swede

**Reply by mriscoc** [reply]
In previous versions the bed was forced to move to front after the XY home, but that behaviour was sometimes inconsistent due to queue abort and stepper drivers being disabled.
